test/core/client_channel/lb_policy/pick_first_test.cc

```diff
@@ -114,11 +112,8 @@ class PickFirstTest : public LoadBalancingPolicyTest {
         // Then it should become disconnected.
         subchannel->SetConnectivityState(GRPC_CHANNEL_IDLE);
         ExpectReresolutionRequest();
-        if (exit_idle_after_connection_fails) {
-          ExpectStateAndQueuingPicker(GRPC_CHANNEL_IDLE);
-        } else {
-          ExpectState(GRPC_CHANNEL_IDLE);
-        }
+        // Do not trigger exit idle.
```

Comment:
This comment isn't sufficiently clear.  I suggest the following wording:

"""
We would normally call ExpectStateAndQueueingPicker() here instead of just ExpectState().  However, calling the picker would also trigger exiting IDLE, which we don't want here, because if the test is going to send an address list update and call GetOrderAddressesArePicked() again, we don't want to trigger a connection attempt on any subchannel until after that next address list update is processed.
"""